EnglishTranslation-master/ETC.tsv

```diff
@@ -6335,67 +6335,67 @@ ETC_20150317_006335	Salavija Hill
 ETC_20150317_006336	Abandoned Camp
 ETC_20150317_006337	Hollow Tree Forest
 ETC_20150317_006338	Field of Reeds
-ETC_20150317_006339	Ishiblysh Square
+ETC_20150317_006339	Isblyskes Square
 ETC_20150317_006340	Ascetic Sanctuary
 ETC_20150317_006341	Nevasingas Highway
-ETC_20150317_006342	Kabeti Hill
+ETC_20150317_006342	Kabantis Hill
 ETC_20150317_006343	Nomads Camp
-ETC_20150317_006344	Zakole Shore
+ETC_20150317_006344	Pusmenulio Shore
```

Comment:
Pusmenulis' (Crescent's Pass) if this should be a possessive form. If not, just Pusmenulis (Crescent Pass)